### DISCUSSION TITLE
Bash guide?

### DISCUSSION THREAD

**Discussion by kaihendry** [Homebrew/discussions#4050] [Everyday usage] [answered]
I'm stuck install bash on a new m1pro https://gist.github.com/kaihendry/fae13df3b84c996acaa0778b019d51fd

Online docs suggest `/usr/local/bin/bash` but it's not setup after installing?

**Comment by p-linnane** [comment, marked as answer]
Homebrew is installed to `/opt/homebrew/` on Apple Silicon, not `/usr/local/`.
